md/script/evalCodeBlocks.ts

```diff
@@ -25,27 +25,33 @@ export async function evalCodeBlocks(
 
     const { lang, code } = details;
 
-    const langLength = lang?.length ?? 0;
-    const inlineCode = code.trim().replace(/\n/g, "\\n");
-    const firstChars = inlineCode.slice(0, consoleWidth - langLength - 4);
+    const icon = (result instanceof NoLanguageError ||
+        result instanceof UnknownLanguageError)
+      ? "skip"
+      : (result instanceof Error || !result.success)
+      ? "✗"
+      : "✔️";
+
+    const colorIcon = icon === "✗"
+      ? red(icon)
+      : icon === "✔️"
+      ? green(icon)
+      : gray(icon);
+
+    const iconLength = icon === "skip" ? 4 : 1;
+    const langLength = String(lang).length;
+
+    const inlineCode = code.trim().replace(/\r?\n/g, "\\n");
+    const firstChars = inlineCode.slice(
+      0,
+      consoleWidth - langLength - iconLength - 5,
+    );
 
     const message = firstChars.length < inlineCode.length
-      ? `${details.lang} ${firstChars}...`
-      : `${details.lang} ${firstChars}`;
+      ? `${colorIcon} ${details.lang} ${firstChars}...`
+      : `${colorIcon} ${details.lang} ${firstChars}`;
```

Comment:
New result output:

<img width="385" alt="Screen Shot 2023-05-06 at 2 23 21 PM" src="https://user-images.githubusercontent.com/47710574/236647428-62ba6f74-f55e-434b-88c6-fa32d266eb58.png">

Reply:
Should I still preserve the `\n` newline characters?

Reply:
> Should I still preserve the `\n` newline characters?

Let's not show the `\n`'s - I don't see a reason to indicate formatting here when the concern is the function of the code. Should we fold it in to this PR or create a TODO?

Reply:
I'll fold it in 👍  Thanks for the input!

Reply:
I went even further with my edit. Now all whitespace is collapsed to a single space, eliminating newlines but also erasing indentation. Here's the updated output at the same width as the previous screenshot.

<img width="387" alt="Screen Shot 2023-05-07 at 7 21 46 PM" src="https://user-images.githubusercontent.com/47710574/236720335-890e1950-64d0-4822-8f2e-2bb6bbf338e9.png">

Reply:
Excellent!